src/include/rados/librados.h

```diff
@@ -1453,6 +1459,42 @@ CEPH_RADOS_API int rados_read(rados_ioctx_t io, const char *oid, char *buf,
                               size_t len, uint64_t off);
 
 /**
+ * Compute checksum from object data
+ *
+ * The io context determines the snapshot to checksum, if any was set
+ * by rados_ioctx_snap_set_read(). The length of the init_value and
+ * resulting checksum are dependent upon the checksum type:
+ *
+ *    XXHASH64: le64
+ *    XXHASH32: le32
+ *    CRC32C:	le32
+ *
+ * The checksum result is encoded the following manner:
+ *
+ *    le32 num_checksum_chunks
+ *    {
+ *    	checksum-type checksum	checksum result for chunk
```

Comment:
"checksum-type checksum	checksum result for chunk" is a bit confusing.  we probably also want to specify that it is little-endian word order.

how about "leXX  checksum for chunk (where XX = appropriate size for the checksum type)"